cadc-vosi/src/main/java/ca/nrc/cadc/vosi/avail/CheckDataSource.java

```diff
@@ -168,6 +188,20 @@ public void check()
         }
         finally
         {
+            if (this.rollback && con != null)
+            {
+                try
+                {
+                    con.rollback();
+                    con.setAutoCommit(true);
```

Comment:
The autoCommit was set to false earlier. So to be complete, just revert it to its original state.